aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -318,9 +328,16 @@ def pg_lang(self) -> List[Dict[str, Any]]:
 
     @property
     def pg_roles(self) -> Dict[str, PGRole]:
+        filtered_roles = ["rdstopmgr"]
```

Comment:
this is to skip default role on aws rds and aurora (which is not needed) that may break the migration process without the previous commit fix.